src/main/java/github/scarsz/discordsrv/listeners/DiscordChatListener.java

```diff
@@ -407,10 +407,11 @@ private boolean processPlayerListCommand(GuildMessageReceivedEvent event, String
         if (!DiscordSRV.config().getBoolean("DiscordChatChannelListCommandEnabled")) return false;
         if (!StringUtils.trimToEmpty(message).equalsIgnoreCase(DiscordSRV.config().getString("DiscordChatChannelListCommandMessage"))) return false;
 
+        int expiration = DiscordSRV.config().getInt("DiscordChatChannelListCommandExpiration") * 1000;
+        String playerListMessage = "";
```

Comment:
Is the initial value really necessary?

Reply:
Not really, it was just left there from before I made any changes.